get.sh

```diff
@@ -660,8 +659,13 @@ getFunctionalTestMaterial()
 	else
 		mv openj9/test/functional functional
 	fi
-
-	rm -rf openj9
+   	
+	cd openj9
+	mkdir -p ../git-backup
+	mv .git ../git-backup
+	rm -rf *
+	mv ../git-backup .git
```

Comment:
I do not think we should move `.git` file around. Can we move it into the subfolder? In this case, it should be `functional`.

Reply:
`.git` is not moved around. `.git` keeps in openj9/.git and all others under openj9 are removed. Same for all available vendors repo as vendor*/.git. 
`aqa-tests/functional` may also have other vendor's test material. For example eclispe adoptium https://github.com/adoptium/temurin-build/tree/master/test/functional/ so it's not good to move .git to `aqa-tests/functional`

Reply:
To delete everything but .git, I would prefer to use git cmd https://stackoverflow.com/questions/22339837/linux-command-to-delete-all-files-except-git-folder

```
git rm -rf .
git clean -fxd
``` 

also, `rm -rf *` is dangerous to use.

Reply:
Updated.